### ISSUE TITLE
[generate:plugin:annotation] namespace is incorrect

### ISSUE THREAD

**Issue by wizonesolutions** [hechoendrupal/DrupalConsole#952]
This is what I did (I'm on Drupal Console 0.9.1):

````
✝ fpd8 ✝ /var/www/site/docroot $ drupal generate:plugin:type:annotation

 Welcome to the Drupal Plugin Type generator 

Enter the module name: fillpdf
Plugin type class name [ExamplePlugin]: FillPdfActionPlugin
Plugin type machine name [fill_pdf_action_plugin]: fillpdf_action
Plugin type label [Fill pdf action plugin]: FillPDF action plugin

 Generated or updated files 

Site path: /var/www/site/docroot
1 - modules/fillpdf/src/Annotation/FillPdfActionPlugin.php
2 - modules/fillpdf/src/Plugin/FillPdfActionPluginBase.php
3 - modules/fillpdf/src/Plugin/FillPdfActionPluginInterface.php
4 - modules/fillpdf/src/Plugin/FillPdfActionPluginManager.php
5 - modules/fillpdf/fillpdf.services.yml
````

The files are generated in `src/Plugin`, but they have a namespace of `Drupal\fillpdf`, not `Drupal\fillpdf\Plugin`. The namespace doesn't match the directory, and PSR-4 won't work.

Also, in `fillpdf.services.yml`, this was added:

````yaml
  plugin.manager.fillpdf_action.processor:
    class: Drupal\fillpdf\Plugin\FillPdfActionPluginManager
    parent: default_plugin_manager
````

The problem is the aforementioned namespace issue.

I can fix the namespaces manually, but without PhpStorm's intelligence telling me that it couldn't resolve `FillPdfActionPluginManager` this would have thrown me off for a while.

**Comment by wizonesolutions** [comment]
Also worth mentioning that every other namespace reference (e.g. in the plugin manager's constructor) seems to be OK and includes the `Plugin` part.

**Comment by jmolivas** [comment]
I will test this and reply with more info.

**Comment by darol100** [comment]
I create a PR (https://github.com/hechoendrupal/DrupalConsole/pull/1742) base on  @sanduhrs changes commit. So we can review the PR later on.

**Comment by jmolivas** [comment]
Fixed on PR #1742
